src/lib/domain/qualifications.ts

```diff
@@ -1,153 +1,35 @@
-// A qualification is something the user has or is.
-// These can later be used to match against Criteria to get points.
-
 import { z } from 'zod'
 
-// TODO: we need to rename the bonus points
-// Qualifications are linked to a Category and a whole category should be scored together.
-// Each visa type has slight variations of what categories are available.
-// Forms are also generated based on these types.
-export const CategorySchema = z.enum([
-  'academic-background',
-  'career',
-  'age',
-  'compensation',
-  'research-achievements',
-  'position',
-  'licenses',
-  'bonus',
-  'contracting-organization',
-  'japanese',
-  'university',
-  'investor',
-])
-
-export type Category = z.infer<typeof CategorySchema>
-
-const BaseQualificationSchema = z.object({
-  category: CategorySchema,
-  id: z.string(),
+export const QualificationsSchema = z.object({
+  degree: z
+    .enum(['doctor', 'master', 'bachelor', 'business_management', 'none'])
+    .optional(),
+  dual_degree: z.boolean().optional(),
+  experience: z.number().optional(), // years of relevant prefessional experience
+  salary: z.number().optional(), // in yen, only counting that from your main source of income
+  age: z.number().optional(), // in years
+  representative_director: z.boolean().optional(),
+  executive_officer: z.boolean().optional(),
+  high_rnd_expenses: z.boolean().optional(), // prev: rnd_exceeds_three_percent
+  foreign_qualification: z.boolean().optional(), // prev: foreign_work_related_qualification
+  growth_field: z.boolean().optional(), // prev: advanced_project_growth_field
+  training_jica: z.boolean().optional(), // prev: completed_training_conducted_by_jica_innovative_asia_project
+  high_investment: z.boolean().optional(), // prev: invested_over_100_million_yen_in_japan
+  investment_management: z.boolean().optional(), // prev: investment_management_business
```

Comment:
why don't we separate the categories through files now in order to make this less difficult to read

Reply:
example, we could do a_education.ts etc.

Reply:
We'll still need the generic type of all qualifications. It would be possible to make Qualifications a union type of `QualificationsEngineering | QualificationsResearcher | QualificationsBusinessManager` but that would make things more complex. For the time being I think it'd be ok to have them merged? I'll revisit as I finish the refactor with the existing pattern.